src/index.html

```diff
@@ -11,7 +11,241 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <!-- <h1>HTML Form</h1>
+    <script type="text/javascript" src="./main.js"></script> -->
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="fieldset">
+        <legend>
+          Personal information
+        </legend>
+
+        <label for="surname" class="fieldsetField">
+          Surname:
+
+          <input
```

Comment:
Also, these blank lines after text are also redundant:
```suggestion
          Surname:
          <input
```